src/components/UserProfile/BlueSquareLayout.jsx

```diff
@@ -26,7 +35,7 @@ const BlueSquareLayout = props => {
         </div>
 
         <BlueSquare
-          blueSquares={infringements}
+          blueSquares={userProfile?.infringments || userProfile?.infringements}
```

Comment:
Looks good as a temporary solution. We will try to fix the misspellings from here to make sure it works.